x/ccv/consumer/keeper/genesis_test.go

```diff
@@ -188,7 +185,7 @@ func TestExportGenesis(t *testing.T) {
 			name: "export a new chain",
 			malleate: func(ctx sdk.Context, ck consumerkeeper.Keeper, mocks testutil.MockedKeepers) {
 				// populate the states used by a new consumer chain
-				cVal, err := consumertypes.NewCCValidator(validator.Address.Bytes(), 1, pubKey)
+				cVal, err := consumertypes.NewCCValidator(validator.Address.Bytes(), 1, cId.SDKPubKey())
```

Comment:
There are several instances of `validator.Address.Bytes()` I think they could be `cId.ABCIAddressBytes()`

Reply:
Even if it's true it might not be worth changing them.